packages/odyssey-react/src/components/Toast/Toast.module.scss

```diff
@@ -82,6 +82,7 @@
   inset-inline-end: $spacing-xs;
   margin-block-end: 0;
   border: 0;
+  line-height: 1;
```

Comment:
the icon was too small here? Looking forward to having this CSS encapsulated within Icon :soon: